lib/vanagon/platform/windows.rb

```diff
@@ -70,6 +70,9 @@ def generate_nuget_packaging_artifacts(workdir, name, binding)
         ["chocolateyInstall.ps1", "chocolateyUninstall.ps1"].each do |win_file|
           FileUtils.copy(File.join(VANAGON_ROOT, "resources/windows/nuget/#{win_file}"), File.join(workdir, win_file))
         end
+
+        # MSI artifacts
+        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.wxs.erb"), File.join(workdir, "#{name}.wxs"), false, { :binding => binding })
```

Comment:
Also, the second argument should be `File.join(workdir, "wix", "#{name}.wxs")` now that we're utilizing a wix subdirectory